manual_scripts/logs/download-cloudfront-logs.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+# Script to download CloudFront logs from S3
+
+DLUHC_ENV=staging
+APP=cpm
+ACCOUNT_ID=486283582667
+PREFIX="ECFDSITRIBUTIONIDHERE.2024-02-16-1"
+
+FOLDER="${APP}-${DLUHC_ENV}-cf-logs"
```

Comment:
Maybe add something to gitignore for the generated files?

`/manual_scripts/logs/*.log` ?